neuralmonkey/dataset.py

```diff
@@ -474,13 +476,19 @@ def maybe_get_series(self, name: str) -> Optional[Iterator]:
             return self.get_series(name)
         return None
 
-    def batches(self, batch_size: int) -> Iterator["Dataset"]:
+    # pylint: disable=too-many-locals,too-many-branches
+    def batches(self,
+                batch_size: int,
+                bucket_span: int = -1,
+                token_level: bool = False) -> Iterator["Dataset"]:
         """Split the dataset into batches.
 
         Arguments:
             batch_size: The size of a batch. In case of lazy datasets, this
                 should be lower than the dataset buffer size. Otherwise, the
                 batch size will be equal to the size of the buffer.
+            bucket_span: Max length span between the buckets.
+            token_level: Compute the batch_size as a number of tokens.
```

Comment:
batch_size se nepočítá (compute), batch_size tu máš danou a maximálně jí můžeš regardovat nebo interpretovat jako počet tokenů. plus by se mělo klarifikovat, že jinak je to počet vět.